src/components/oculus-touch-controls.js

```diff
@@ -220,25 +224,25 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     if (!data.model) { return; }
     // Set the controller display model based on the data passed in.
     this.displayModel = CONTROLLER_PROPERTIES[data.controllerType] || CONTROLLER_PROPERTIES[CONTROLLER_DEFAULT];
-    // If the developer is asking for auto-detection, see if the displayName can be retrieved to identify the specific unit.
-    // This only works for WebVR currently.
```

Comment:
this comment is gone

Reply:
1. This does not only work for WebVR, we do it for both WebVR and WebXR. This comment was out of date.
2. This comment _is_ still in the code: `     // If the developer is asking for auto-detection, use the retrieved displayName to identify the specific unit.`, line 227